README.md

```diff
@@ -119,23 +119,57 @@ cargo build -p millau-bridge-node
 cargo build -p substrate-relay
 ```
 
-### Running
+### Running dev network
+
+We will only launch a dev network to demonstrate the unidirectional relay from millau to rialto here
```

Comment:
This isn't entirely correct. We need a bi-directional bridge in order to relay messages